pkg/minikube/image/image.go

```diff
@@ -94,6 +96,26 @@ func ExistsImageInDaemon(img string) bool {
 	return false
 }
 
+// LoadFromTarball checks if the image exists as a tarball and tries to load it to the local daemon
+// TODO: Pass in if we are loading to docker or podman so this function can also be used for podman
```

Comment:
for simplicity, instead of this TODO. you can pass in the dameon-type and then for non-docker ones return "not implemented yet. see isssue #1234" error

Reply:
SGTM, done!